r2/r2/public/static/css/highlight.css

```diff
@@ -8,7 +8,7 @@
 }
 
 /* github.com style (c) Vasily Polovnyov <vast@whiteants.net> - from the highlight.js source */
-pre {
+pre.subreddit-stylesheet-source {
```

Comment:
Nuance due to how this style would affect the pre elements showing the image urls.